schemas/triniti/canvas/mixin/iframe-block/latest.xml

```diff
@@ -10,6 +10,9 @@
       <field name="width" type="string" pattern="^\d+(%|px)?$"/>
```

Comment:
@nanetteCodes - bump the version number (similar to this: https://github.com/triniti/schemas/commit/15e1a9624dfff97131ab72d5e21a5fb0f517d318#diff-4867afa1ab1e20144b7e88cdaf4ff158R6) for the schemas and run `composer pbjc` again,